src/Nethermind/Nethermind.Core/Transaction.cs

```diff
@@ -52,6 +52,8 @@ public class Transaction
         public bool SupportsBlobs => Type.SupportsBlobs();
         public bool SupportsAuthorizationList => Type.SupportsAuthorizationList();
         public long GasLimit { get; set; }
+        [JsonIgnore]
+        public long? SpentGas { get; set; }
```

Comment:
We always fallback on null with `??`. Could we make it long maybe? One less `?`

Reply:
Made non nullable